app/Repositories/LocationRepository.php

```diff
@@ -41,6 +42,19 @@ public function getByWorkspace($workspaceId)
             ->toArray();
     }
 
+    public function getAllLocation()
+    {
+        $data = [];
+        $locations = $this->model->all();
+
+        foreach ($locations as $location) {
+            $workspace = Workspace::findOrFail($location->workspace_id);
+            $data[$location->id] = $workspace->name . '(' . $location->name . ')'; 
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Squiz.WhiteSpace.SuperfluousWhitespace.EndLine</br>**Severity**: error</br>**File**: app/Repositories/LocationRepository.php</br>**Line**: 52</br>Whitespace found at end of line